custom/icds_reports/utils/__init__.py

```diff
@@ -431,14 +431,14 @@ def get_location_filter(location_id, domain, include_object=False):
         sql_location = SQLLocation.objects.get(location_id=location_id, domain=domain)
     except SQLLocation.DoesNotExist:
         return {'aggregation_level': 1}
-    config.update(
-        {
-            ('%s_id' % ancestor.location_type.code): ancestor.location_id
-            for ancestor in sql_location.get_ancestors(include_self=True)
-        }
-    )
-    config['aggregation_level'] = len(config) + 1
 
+    location_ids = {}
```

Comment:
this change is to address https://github.com/dimagi/commcare-hq/pull/27687#discussion_r436435420